frontend/playwright.config.ts

```diff
@@ -114,8 +116,8 @@ const config: PlaywrightTestConfig = {
 
    /* Run your local dev server before starting the tests */
    webServer: {
-      cwd: '../',
-      command: 'pnpm run dev',
+      cwd: process.env.CI ? './' : '../',
+      command: process.env.CI ? 'pnpm run build && pnpm start' : 'pnpm dev',
```

Comment:
Should this be

`command: process.env.CI ? 'pnpm run build && pnpm start' : 'pnpm run dev',`

Reply:
I think `pnpm run dev` vs `pnpm dev` are basically the same. From https://pnpm.io/cli/run#examples: 

> Let's say you have a watch script configured in your package.json, like so:
> ```
>"scripts": {
>    "watch": "webpack --watch"
>}
>```
> You can now run that script by using pnpm run watch! Simple, right? Another thing to note for those that like to save keystrokes and time is that all scripts get aliased in as pnpm commands, so ultimately pnpm watch is just shorthand for pnpm run watch (ONLY for scripts that do not share the same name as already existing pnpm commands).

Reply:
Ah, @batbattur we should make this consistent so there is no confusion. Let's remove the `run` part for `pnpm run build`


@mlahargou it is basically the same thing for `pnpm`

> Another thing to note for those that like to save keystrokes and time is that all scripts get aliased in as pnpm commands, so ultimately pnpm watch is just shorthand for pnpm run watch (ONLY for scripts that do not share the same name as already existing pnpm commands).

https://pnpm.io/cli/run

Reply:
Updated in https://github.com/iFixit/react-commerce/pull/1193/commits/668c528b9ec44fa65d010d32203238d81bcc9476.